Sources/StreamChatUI/UIConfig+Images.swift

```diff
@@ -0,0 +1,197 @@
+        public var fileRtf: UIImage = UIImage(named: "rtf", in: .streamChatUI)!
+        public var fileTargz: UIImage = UIImage(named: "tar.gz", in: .streamChatUI)!
+        public var fileTxt: UIImage = UIImage(named: "txt", in: .streamChatUI)!
+        public var fileXls: UIImage = UIImage(named: "xls", in: .streamChatUI)!
+        public var fileXlsx: UIImage = UIImage(named: "xlsx", in: .streamChatUI)!
+        public var filezip: UIImage = UIImage(named: "zip", in: .streamChatUI)!
+        public var fileFallback: UIImage = UIImage(named: "generic", in: .streamChatUI)!
+
+        private var _documentPreviews: [String: UIImage]?
+
+        public var documentPreviews: [String: UIImage] {
+            get { _documentPreviews ??
+                [
+                    "7z": file7z,
+                    "csv": fileCsv,
+                    "doc": fileDoc,
+                    "docx": fileDocx,
+                    "html": fileHtml,
+                    "md": fileMd,
+                    "mp3": fileMp3,
+                    "odt": fileOdt,
+                    "pdf": filePdf,
+                    "ppt": filePpt,
+                    "pptx": filePptx,
+                    "rar": fileRar,
+                    "rtf": fileRtf,
+                    "tar.gz": fileTargz,
+                    "txt": fileTxt,
+                    "xls": fileXls,
+                    "xlsx": fileXlsx,
+                    "zip": filezip
+                ]
+            }
+            set { _documentPreviews = newValue }
+        }
+
+        // MARK: - Message Actions
+
+        public var messageActionInlineReply: UIImage = UIImage(named: "icn_inline_reply", in: .streamChatUI)!
+        public var messageActionThreadReply: UIImage = UIImage(named: "icn_thread_reply", in: .streamChatUI)!
+        public var messageActionEdit: UIImage = UIImage(named: "icn_edit", in: .streamChatUI)!
+        public var messageActionCopy: UIImage = UIImage(named: "icn_copy", in: .streamChatUI)!
+        public var messageActionBlockUser: UIImage = UIImage(named: "icn_block_user", in: .streamChatUI)!
+        public var messageActionMuteUser: UIImage = UIImage(named: "icn_mute_user", in: .streamChatUI)!
+        public var messageActionDelete: UIImage = UIImage(named: "icn_delete", in: .streamChatUI)!
+        public var messageActionResend: UIImage = UIImage(named: "icn_resend", in: .streamChatUI)!
+
+        // MARK: - Placeholders
+
+        public var userAvatarPlaceholder1: UIImage = UIImage(named: "pattern1", in: .streamChatUI)!
+        public var userAvatarPlaceholder2: UIImage = UIImage(named: "pattern2", in: .streamChatUI)!
+        public var userAvatarPlaceholder3: UIImage = UIImage(named: "pattern3", in: .streamChatUI)!
+        public var userAvatarPlaceholder4: UIImage = UIImage(named: "pattern4", in: .streamChatUI)!
+        public var userAvatarPlaceholder5: UIImage = UIImage(named: "pattern5", in: .streamChatUI)!
```

Comment:
Just like we have the views separated by structs, can't we have the images too? Like `Image.files`, `Image.reactions`, `Images.messageAction`. Just to avoid having a Huge `Images` struct.

Reply:
I see what you mean @nuno-vieira but I'd suggest a small modification. Since this should serve as the main pool of images you can use across the app, rather than organizing it by "where it's used" it can be organized by "what is it". So like `Images.icons`, `Images.reactions`, or simply just `Images.backButton` for images without caterogy.